### DISCUSSION TITLE
Maximum forecast horizon

### DISCUSSION THREAD

**Discussion by lneumanntwe** [autogluon/autogluon#4316] [Q&A]
I have a relatively short timeseries of available data for training (72 months/6 years) for which I need to generate a forest for the next 5 years. Is that possible with autogluon or am I limited to a prediction length = length(train_data) -1? If I tried a prediction length above or equal to 36 I get

"Provided train_data has 72 rows, 1 time series. Median time series length is 72 (min=72, max=72). 
Removing 1 short time series from train_data. Only series with length >= 73 will be used for training.
After filtering, train_data has 0 rows, 0 time series. Median time series length is nan (min=nan, max=nan)"

**Comment by shchur** [comment]
Hi @lneumanntwe, AutoGluon performs model selection internally, which requires to reserve some data for validation. Therefore, it is required that at least some time series have length `>= 2 * prediction_length + 1` (see [documentation](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-indepth.html#what-data-format-is-expected-by-timeseriespredictor)).

If you only want to train a single model and don't want to perform model selection, you can use the `skip_model_selection` argument, e.g.:
```python
predictor = TimeSeriesPredictor(...).fit(train_data, hyperparameters={"ETS": {}}, skip_model_selection=True)
```
In this case your training time series must have at least `>= prediction_length + 1` observations.
